docs/vocabulary.md

```diff
@@ -388,6 +389,230 @@ pipeline in corresponding documentation.
+* Other Results and Side Effects: May also update the Source Code Repository with new dependencies, create a pull request with the updates, or open an issue requesting the updates.
+
+#### Test
+* Semantics: Run a test suite. Examples includes unit tests, integration tests, acceptance tests, performance tests, canary tests, A/B tests, smoke tests.
+* Aliases: Validate
+* Inputs: Any
+* Outputs: Test Results, Test Reports, Test Coverage Reports
+* Other Results and Side Effects: N/A
+
+#### Coverage
+* Semantics: Verifies that the testing coverage meets a standard.
+* Aliases: Code Coverage, Test Coverage
+* Inputs: Test Coverage Reports 
+* Outputs: Coverage Verification Report
+* Other Results and Side Effects: N/A
+
+#### Scan
+* Semantics: Use a tool to do verification of software and documenation other than testing. Examples include static code analysis, linting, checking for known vulnerabilities in code or binaries, dynamic security scans, license checks, and code smells.
+* Aliases: Check
+* Inputs: Software Source, Binary Source, Generated Software, Generated Binaries, Dependency List/Graph
+* Outputs: Scan Results, Scan Reports, Scan Coverage Reports
+* Other Results and Side Effects: N/A
+
+#### Bill of Materials
+* Semantics: Create a Software Bill of Materials (SBoM) for a given repository that captures pedigree of all the dependencies and is collected at different granularities.
+* Aliases: BOM, SBOM
+* Inputs: Software Source, Binary Source, Generated Software, Generated Binaries, Dependency List/Graph, Packaged Artifacts
+* Outputs: Build BOM
+* Other Results and Side Effects: N/A
+
+#### Package
+* Semantics: Create the software artifact(s) that will be published. Typically updates the software version.
+* Aliases: Containerize
+* Inputs: Software Source, Binary Source, Generated Software, Generated Binaries, Dependency List/Graph
+* Outputs: Packaged Artifacts (Software, Documentation or Binaries), Container Images, Image Tags, Image Digests, Archives
+* Other Results and Side Effects: N/A
+
+#### Sign
+* Semantics: Use a cryptographic method to authenticate the software. The signature may also include information about the source of the software, how or where it was built, and what level of approval it has received (staging, production, etc.).
+* Aliases: N/A
+* Inputs: Packaged Artifacts
+* Outputs: Signed Artifacts, Signing Record(s)
+* Other Results and Side Effects: N/A
+
+#### Policy
+* Semantics: Verifies that policies are followed, for example:  software is from a trusted source, source repositories are configured correctly, Kubernetes manifests are configured securely, dependencies are signed, code standards are followed, code reviews are completed, there is a secure chain of custody, or appropriate work items or change requests are associated with the change.
+* Aliases: Check, Gate, Provenance
+* Inputs: Any
+* Outputs: Policy and/or Provenance Reports
```

Comment:
wouldn't decisions such as blocking a PR/commit to be accepted, artifact/image to be discarded/rolled out be considered as outputs as well in case there is a violation of policy?

Reply:
I think those are really side effects, so I put these examples in the Side Effects list.